### PR TITLE
Feature branch samuel

### PR DESCRIPTION
**Pull request by SamuelTTW**
Structured page with HTML/CSS/JS Which was made for practice in the pre-Enrollment-Bootcamp, With .html file, called index, .css file called Style.css a .js file called  main.js

### CODE REVIEW

**Review by SamuelTTW — COMMENTED** [review]
Hello, I see that you were doing the practice of the Odin Project, that is very good, however I kindly recommend some changes in the structure of the page

File: index.html [hunk 1]
```diff
@@ -35,9 +35,9 @@
             </div>
 
             <div class="Profile">
-                <button type="menu" id="Video-Upload"><i class="fa-solid fa-upload"></i></button>
-                <button type="menu" id="Tabloide"><i class="fa-solid fa-table-cells"></i></button>
-                <button type="menu" id="Bell"><i class="fa-solid fa-bell"></i></button>
```
Comment (SamuelTTW) [inline index.html:40]: I kindly recommend that you assign the type of button that it is, in this case, it would be type "menu"

File: index.html [hunk 2]
```diff
@@ -35,21 +35,23 @@
...
         </header>
 
         <nav class="menu-lateral">
+
             <a href="#" class="active"><i class="fa-solid fa-house"></i>Inicio</a>
             <a href="#"><i class="fa-solid fa-fire-flame-curved"></i>Tendencias</a>
             <a href="#"><i class="fa-solid fa-star"></i>Suscripciones</a>
             <a href="#"><i class="fa-solid fa-folder"></i>Element</a>
             <a href="#"><i class="fa-solid fa-folder"></i>Element2</a>
             <hr>
```
Comment (SamuelTTW) [inline index.html:53]: You can improve the presentation by writing the sections so that they don't just say "Element"

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Almost There! 🏃

## Hi @SamuelTTW 👋

## Welcome to Microverse! 🌎

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃

## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Especially if that's is your first project 1️⃣. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better job** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights 🌟

- ✔️ good use of inline comments
- ✔️ added more than 2 comments

## Required Changes ♻️

- [ ] [REQUIRED] Please, make sure to add a comment about the PR Description. Since it is not there

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @SamuelTTW 👋

Your project is complete! ✔️ 🏁. This is the First Project 1️⃣ You get Approved ✔️! in microverse 🌐.

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**